pkg/agent/params.go

```diff
@@ -103,9 +104,21 @@ func assignKubernetesParametersFromAgentProfile(profile *datamodel.AgentPoolProf
 	}
 }
 
-func assignKubernetesParametersfromKubernetesConfig(kubernetesConfig *datamodel.KubernetesConfig, properties *datamodel.Properties,
-	parametersMap paramsMap, cloudSpecConfig *datamodel.AzureEnvironmentSpecConfig, k8sComponents *datamodel.K8sComponents,
-	config *datamodel.NodeBootstrappingConfiguration, k8sVersion string) {
+func assignKubernetesParametersfromKubernetesConfig(properties *datamodel.Properties, parametersMap paramsMap,
+	cloudSpecConfig *datamodel.AzureEnvironmentSpecConfig,
+	k8sComponents *datamodel.K8sComponents,
+	config *datamodel.NodeBootstrappingConfiguration) {
+	orchestratorProfile := properties.OrchestratorProfile
+
+	if !orchestratorProfile.IsKubernetes() {
```

Comment:
nit: new logic? or am i misreading the translation?

Reply:
I don't think there is any new logic here. I just removed unused parameters and added assignKubernetesParametersfromKubernetesConfig to lower the complecity of assignKubernetesParameters as well as restructured it a bit to get rid of an if statment.